Services/communication_service.c

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (C) 2015  University of Alberta
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ */
+/**
+ * @file communication_service.c
+ * @author Arash Yazdani
+ * @date 2020-09-25
+ */
+
+#include "communication_service.h"
+
+SAT_returnState communication_service_app(csp_packet_t *packet) {
```

Comment:
@ArashYzd I know the other services don't, but could you add a block comment for the function following the format of the other functions? Just make a note that it takes a CSP packet and switches based on the subservice command

Reply:
Sure.

Reply:
I'll merge once you do :+1: